### ISSUE TITLE
typescript: missing semicolons after export declare function

### ISSUE THREAD

**Issue by rossipedia** [prettier/prettier#2598]
Steps to reproduce:
``` bash
$ npm install --global prettier
$ echo 'export declare function foobar(): int;' | prettier --stdin --parser typescript
export declare function foobar(): int
```
Expected output:
```
                                     ↓
export declare function foobar(): int;
```

Prettier version: 1.5.3

**Comment by ikatyang** [comment]
Should be fixed by #2536.

**Comment by rossipedia** [comment]
Awesome! Thanks for the prompt fix 👍
